app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -159,7 +166,7 @@ public void refresh(Detail detail, TreeReference reference, int index, boolean h
                 hasDetailCalloutListener, new DefaultEDVModifier()
         );
         mViewPager.setAdapter(mEntityDetailPagerAdapter);
-        markSelectedTab(0);
+        if(!useNewTabStyle) markSelectedTab(0);

```

Comment:
removed the useNewTabStyle var, it makes no sense at all (we'll always want to use the new style, right?)